### ISSUE TITLE
About the training data

### ISSUE THREAD

**Issue by mariolew** [FalongShen/styletransfer#12]
The paper said the test set of WikiArt, which contains 80k images is used. However, the test set of WikiArt does not contain 80k images, but the train set does contain 80k images. I wonder if you use the train set of WikiArt?

**Comment by shenfalong** [comment]
Thanks, you are right.
